app/src/common/shared/com/igalia/wolvic/downloads/DownloadsManager.java

```diff
@@ -213,6 +218,9 @@ public void downloadBlobUri(DownloadJob job) {
         }
         Log.i(LOGTAG, "Saved " + job.getUri() + " to " + file.getName() + " (" + readBytes + " bytes)");
 
+        // TODO: This method was deprecated in API level 29.
+        //  Apps should instead contribute files to MediaStore.Downloads collection to make them
+        //  available to user as part of Downloads.
```

Comment:
This is a nice finding, mind adding a new bug for this and include the link in this comment?

Doing that will allow us to get rid of all the nasty code we have to copy files from wolvic's storage data to system's

Reply:
https://github.com/Igalia/wolvic/issues/798